From bb0ec0c9e8996801d5925831d9b2001d76c1cff2 Mon Sep 17 00:00:00 2001
From: Robert Church <robert.church@windriver.com>
Date: Mon, 6 Jun 2022 01:06:15 -0400
Subject: [PATCH] Change from unsupported gitarchive format to quilt

The current build infrastructure doesn't support source patching of
archives that have the gitarchive source format.

Signed-off-by: Robert Church <robert.church@windriver.com>
---
 debian/source/format | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/source/format b/debian/source/format
index 7cb8623..163aaf8 100644
--- a/debian/source/format
+++ b/debian/source/format
@@ -1 +1 @@
-3.0 (gitarchive)
+3.0 (quilt)
-- 
2.16.6

